src/complement.js

```diff
@@ -3,15 +3,10 @@ var not = require('./not');
 
 
 /**
- * Takes a function `f` and returns a function `g` such that:
+ * Takes a function `f` and returns a function `g` such that if called with the same arguments
+ * when `f` returns `true`, `g` returns `false` and vice versa
  *
- *   - applying `g` to zero or more arguments will give __true__ if applying
- *     the same arguments to `f` gives a logical __false__ value; and
- *
- *   - applying `g` to zero or more arguments will give __false__ if applying
- *     the same arguments to `f` gives a logical __true__ value.
- *
- * `R.complement` will work on all other functors as well.
```

Comment:
I prefer the current wording, but I think I wrote it so it's not surprising it reads well to me. ;)